doc/source/regression/dwt-idwt.md

```diff
@@ -9,41 +9,18 @@ kernelspec:
   display_name: Python 3 (ipykernel)
   language: python
   name: python3
-mystnb:
-  execution_allow_errors: true
-  execution_show_tb: true
 ---
 
 +++ {"tags": ["jupyterlite_sphinx_strip"]}
 
-```{eval-rst}
-.. currentmodule:: pywt
-
-.. dropdown:: 🧑‍🔬 This notebook can be executed online. Click this section to try it out! ✨
-    :color: success
-
-    .. notebooklite:: dwt-idwt.ipynb
-      :width: 100%
-      :height: 600px
-      :prompt: Open notebook
-
-.. dropdown:: Download this notebook
-    :color: info
-    :open:
-
-    Please use the following links to download this notebook in various formats:
-
-    1. :download:`Download IPyNB (IPython Notebook) <dwt-idwt.ipynb>`
-    2. :download:`Download Markdown Notebook (Jupytext) <dwt-idwt.md>`
+```{include} header.md
 ```
 
-+++
-
 # DWT and IDWT
 
 ## Discrete Wavelet Transform
 
-Let's do a Discrete Wavelet Transform of some sample data `x`
+Let's do a [Discrete Wavelet Transform](ref-dwt) of some sample data `x`
```

Comment:
In some places I restored the cross-references even though they won't work in the notebook file because I think they were valuable enough to keep in the docs page even if they don't work in the notebook file. I tried to do this sparingly. 

Honestly, I was a bit surprised to see that this doesn't work. It really seems that one of the tools that converts to ipynb should support this.

Reply:
>  It really seems that one of the tools that converts to ipynb should support this.

I also had the exact same thought over the holidays. I think supporting these kinds of references might be possible, since they are widespread in SciPy's notebooks – where we would remove the references that are caught with a regex pattern and drop HTML links to them if they are found. I shall take a look at it.